scripts/tests/chiptest/test_definition.py

```diff
@@ -239,16 +240,18 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
             # so it will be commissionable again.
             app.factoryReset()
             app.start(str(randrange(1, 4096)))
-
-            runner.RunSubprocess(
-                tool_cmd + ['pairing', 'qrcode', TEST_NODE_ID, app.setupCode] +
-                ['--paa-trust-store-path', DEVELOPMENT_PAA_LIST],
-                name='PAIR', dependencies=[apps_register])
-
+            trust_store_path = [] if sys.platform == 'darwin' else [
+                '--paa-trust-store-path', DEVELOPMENT_PAA_LIST]
+            pairing_cmd = tool_cmd + \
+                ['pairing', 'qrcode', TEST_NODE_ID,
+                    app.setupCode] + trust_store_path
+            runner.RunSubprocess(pairing_cmd,
+                                 name='PAIR', dependencies=[apps_register])
+
+            test_cmd = tool_cmd + ['tests', self.run_name] + \
```

Comment:
does order of arguments matter? otherwise you can add --PICS after run_name and have less lists.

Reply:
I don't this it does. I can update this on the other PR.